chapter-11/end/Engine/MainGame.cs

```diff
@@ -49,6 +44,12 @@ public MainGame(int width, int height, BaseGameState firstGameState)
         /// </summary>
         protected override void Initialize()
         {
+            graphics.PreferredBackBufferWidth = _DesignedResolutionWidth;
+            graphics.PreferredBackBufferHeight = _DesignedResolutionHeight;
+            graphics.IsFullScreen = false;
+            graphics.SynchronizeWithVerticalRetrace = false;
```

Comment:
This was set to False for only this chapter vs others - intentional?

Reply:
Intentional for now because I figured this one out at the end after already changing all 16 solutions :) 

If you agree with this I can go change it everywhere, but you'll have to explain this code in your early chapters. This v-synch setting goes hand in hand with the 2 new game settings in program.cs:

```
game.IsFixedTimeStep = true;
game.TargetElapsedTime = TimeSpan.FromMilliseconds(1000.0f / 60);
```

Reply:
Or, I can mention this code in chapter 11 as a way to improve performance as the screen gets busier, but I think writing about this in the earlier chapters may be better. If I understood things correctly, this code forces the game to use 60 frames per seconds and Monogame will not drop down to 30 fps when things get rough, causing the stutter.

Reply:
there's a good explanation here down in the replies: https://community.monogame.net/t/solved-jittery-choppy-frame-movement-for-2d-game/11888/2

Reply:
All good - I'm good keeping it until Chapter 11